dist/Storable/Storable.xs

```diff
@@ -2597,18 +2620,16 @@ static int store_scalar(pTHX_ stcxt_t *cxt, SV *sv)
     string:
 
 #ifdef SvVOK
-        if (SvMAGICAL(sv) && (mg = mg_find(sv, 'V'))) {
+        if (sv_vstring_get(sv, &vstr_pv, &vstr_len)) {
```

Comment:
If I were a serializing XS mod, and performance is important, `sv_vstring_get` needs to be `#define SvVSTRING(sv, pv, n)` with a fast shortcut on `SvMAGICAL(sv) &&` before executing "heavy" extern exported  PLT/GOT C symbol `Perl_sv_vstring_get()`.